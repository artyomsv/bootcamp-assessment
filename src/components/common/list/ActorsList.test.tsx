import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import ActorsList from './ActorsList.widget';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const mockFn = jest.fn();
    const result = shallow(
      <ActorsList
        actors={[]}
        expanded={-1}
        expand={mockFn}
        navigateToActor={mockFn}
        navigateToMovie={mockFn}
      />
    );
    expect(result).toMatchSnapshot();
  });
});
