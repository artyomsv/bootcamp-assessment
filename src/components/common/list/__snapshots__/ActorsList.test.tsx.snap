// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<ActorsList
  actors={Array []}
  classes={
    Object {
      "column": "ActorsList-column-5",
      "details": "ActorsList-details-4",
      "heading": "ActorsList-heading-3",
      "image": "ActorsList-image-2",
      "root": "ActorsList-root-1",
    }
  }
  expand={[MockFunction]}
  expanded={-1}
  navigateToActor={[MockFunction]}
  navigateToMovie={[MockFunction]}
/>
`;
